target/portfolio-1/js/createGame.js

```diff
@@ -1,12 +1,33 @@
+function initMapCreateGame() {
+  var myLatlng = {lat: 0.0, lng: 0.0};
```

Comment:
(0.0, 0.0) is not an unknown position, it's the center of the world map. As I stated in the comment below, " This is for when the map loads very zoomed out, we want the origin of the map to be the center and that is why I have it (changed since this) to be (0, 0). This is to show that we are very biased toward any country."

Also, just to not repeat my other comment that I have above, wouldn't latLng be more descriptive than position?